wmfdata/hdfs.py

```diff
@@ -0,0 +1,29 @@
+import subprocess
+
+from utils import check_kerberos_auth
+
+def put(local_path, hdfs_path):
+    """
+    Puts the local file located at `local_path` into our Hadoop file system
+    at `hdfs_path`. `hdfs_path` does not need to include the scheme ("hdfs://");
+    relative paths are interpreted relative to your user directory.
+    """
+    check_kerberos_auth()
+    subprocess.run(
+        ["hdfs", "dfs", "-put", local_path, hdfs_path],
```

Comment:
Not for this change, but FYI we are starting to use the [pyarrow HDFS](https://arrow.apache.org/docs/python/filesystems.html#filesystem-hdfs) interface everywhere.  Even better is [fsspec's pyarrow wrapper](https://github.com/fsspec/filesystem_spec/blob/master/fsspec/implementations/arrow.py#L190), allowing to use a pythonic file interface for HDFS, but also for almost any 'filesystem' like URL, including HDFS>

Reply:
Oh, wait, hdfs.py is new?  Maybe you should use fsspec or pyarrow for this change.